docs/developer-guide.md

```diff
@@ -157,6 +157,13 @@ Now you can run `./risedev d` to start a new dev cluster. The new dev cluster wi
 
 You may also add multiple compute nodes in the cluster. The `ci-3cn-1fe` config is an example.
 
+### Configure system variables
+
+You can configure or override system variables in `src/config/risingwave.toml`.
+Not all the variables are explicitly set in this file. 
+You may check `src/common/src/config.rs` to see all the configurable variables,
+and set each one in the corresponding section such as `[server]`, `[storage]`.
```

Comment:
@lmatz Question: If developers want to include additional variables, they need to add variables and values to the .toml file. Right? Does it matter if a variable is added to a particular section vs another one? If not, we can say something like this:
"You can check ... to see all the configurable variables. If additional variables are needed, edit `src/config/risingwave.toml` to include them [in the corresponding section].

Reply:
Yes, they do.
Yes, it matters.

do you mean `if yes, we can say something like this:`?

Reply:
@lmatz Yes. If yes, then here is the proposed wording:
You can check ... to see all the configurable variables. If additional variables are needed, include them in the correct sections (such as `[server]` or `[storage]') in `src/config/risingwave.toml`.